GameServer/Logic/Chatbox/Commands/NewCommand.cs

```diff
@@ -14,25 +14,28 @@ public override void Execute(Peer peer, bool hasReceivedArguments, string argume
         {
             var netid = _owner.GetGame().GetPeerInfo(peer).GetChampion().getNetId();
 
-            var packet = new Packets.BasePacket((PacketCmdS2C)0xB1);
-            var buffer = packet.getBuffer();
-            buffer.Write(Encoding.Default.GetBytes("Right message box edited, new message, a very loooooooooooooooooooooooooooooong one"));
-            buffer.Write(0x00);
+            /*var coso = _owner.GetGame().GimmeCoso();
+            _owner.SendDebugMsgFormatted(DebugMsgType.INFO, "Coso: " + coso);
+            Console.WriteLine(coso.ToString());*/
+            if (true)//!Enum.IsDefined(typeof(ExtendedPacketCmd), (uint)coso))
```

Comment:
What's up with this? Either remove the whole if statement, or add an actual condition